bulk_query_gen/influxdb/influx_iot_multi_measurement_or.go

```diff
@@ -0,0 +1,34 @@
+package influxdb
```

Comment:
This file was renamed from `influx_iot_battery_voltage.go`, with the `BatteryLevels` replaced with `MultiMeasurementOr`